src/Slider.js

```diff
@@ -3,17 +3,61 @@ import PropTypes from 'prop-types';
 import cx from 'classnames';
 
 class Slider extends Component {
-  componentDidMount() {
-    const { fullscreen, indicators, interval, transition } = this.props;
+  constructor() {
+    super();
+    this.initSlider = this.initSlider.bind(this);
+    this.fullscreenReset = this.fullscreenReset.bind(this);
+    this.setActiveIndex = this.setActiveIndex.bind(this);
+  }
 
-    $('.slider').slider({
-      full_width: fullscreen,
+  initSlider() {
+    const { indicators, interval, duration, height } = this.props;
```

Comment:
In other components, we normally pass `options` prop, which is an object that is passed to initialise, please see https://github.com/react-materialize/react-materialize/blob/master/src/Carousel.js#L96 for an example